examples/riscv/test-riscv.sml

```diff
@@ -27,15 +28,34 @@ val _ = print_and_check_thm
   "swap RISC-V lift theorem"
   bir_swap_riscv_lift_THM
   ``
-  bir_is_lifted_prog riscv_bmr (WI_end (0w : word64) (0x1000000w : word64))
+  bir_is_lifted_prog riscv_bmr (WI_end (0w : word64) (32w : word64))
    bir_swap_progbin
    (bir_swap_prog : 'observation_type bir_program_t)
   ``;
 
+val _ = print_and_check_thm
+  "swap BIR contract theorem"
+  bir_cont_swap
+  ``bir_cont (bir_swap_prog : 'observation_type bir_program_t)
+    bir_exp_true (BL_Address (Imm64 0w))
+    {BL_Address (Imm64 20w)} {} bir_swap_pre
+    (\l. if l = BL_Address (Imm64 20w) then bir_swap_post
+        else bir_exp_false)
+  ``;
+
 val _ = print_and_check_thm
   "swap RISC-V backlifted theorem"
   riscv_swap_contract_thm
   ``riscv_cont
      bir_swap_progbin
      0w {20w}
      riscv_swap_pre riscv_swap_post``;
+  
+val _ = print_and_check_thm
+  "incr RISC-V lift theorem"
+  bir_incr_riscv_lift_THM
+  ``
+  bir_is_lifted_prog riscv_bmr (WI_end (0w : word64) (8w : word64))
+   bir_incr_progbin
+   (bir_incr_prog : 'observation_type bir_program_t)
+  ``;
```

Comment:
How about adding a test for the final `incr` theorem? Some intermediate theorem could also make sense?